<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationWidget</class>
 <widget class="QWidget" name="NavigationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1315</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MapCoreWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>1010</width>
         <height>706</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1010</width>
         <height>706</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QComboBox" name="comboBox_1">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>10</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_2">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>70</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>54</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>起点：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>54</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>终点：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>140</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>查询路径</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>路径结果：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_dis">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>最短路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapCoreWidget</class>
   <extends>QWidget</extends>
   <header>mapcorewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
